packages/fxa-auth-server/lib/metrics/glean/index.ts

```diff
@@ -51,30 +52,35 @@ const findUid = (request: MetricsRequest, metricsData?: MetricsData): string =>
 const sha256HashUid = (uid: string) =>
   createHash('sha256').update(uid).digest('hex');
 
-const findOauthClientId = (
-  request: MetricsRequest,
-  metricsData?: MetricsData
-): string =>
-  metricsData?.oauthClientId ||
-  request.auth.credentials?.client_id ||
-  request.payload?.client_id ||
-  '';
-
 const findServiceName = async (request: MetricsRequest) => {
   const metricsContext = await request.app.metricsContext;
+  return metricsContext.service || '';
+};
 
-  if (metricsContext.service) {
-    return metricsContext.service;
-  }
-
-  const clientId = findOauthClientId(request);
-
-  // use the client id to service name mapping from the app config
```

Comment:
The client id is more useful to Data Science and they can look up the actual name of client with the id when querying data.